code/modules/mob/living/damage_procs.dm

```diff
@@ -9,21 +9,24 @@
 	standard 0 if fail
 */
 /mob/living/proc/apply_damage(var/damage = 0,var/damagetype = BRUTE, var/def_zone = null, var/blocked = 0, var/used_weapon = null, var/sharp = 0, var/edge = 0)
```

Comment:
Also, wouldn't you mind cleaning up the code by removing all istances of the blocked var?

Reply:
I'll look into it